Planetary.h

```diff
@@ -0,0 +1,22 @@
+#include "Planet.h"
+#include <bits/stdc++.h>
+class Planetary {
+
+public:
+
+    Planetary();
+    Planetary(int a);
+    ~Planetary();
+
+    void findDistanceBetween(Planet planetA,Planet planetB);
+    void findAverageMass();
+    void SortingPlanets();
+    Planet getPlanetIndex(int Index);
+    void addPlanetToArray(Planet planet,unsigned int Index);
+
+
+private:
+    Planet* ArrayofPlanets;
```

Comment:
No spaces around period or arrow. Pointer operators do not have trailing spaces. Example: Planet *ArrayofPlanets